packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -105,7 +107,7 @@ class ProjectReplace(InputSchema):
     uuid: ProjectID
     name: ShortTruncatedStr
     description: LongTruncatedStr
-    thumbnail: HttpUrl | None
+    thumbnail: Annotated[HttpUrl | None, BeforeValidator(empty_str_to_none_pre_validator)] = Field(default=None)
```

Comment:
nice!

Reply:
All the "external validators" could disappear and write this compact way in these situations.

Reply:
cool :+1: